cdap-elastic/src/test/java/io/cdap/cdap/metadata/elastic/ElasticsearchMetadataStorageTest.java

```diff
@@ -503,6 +515,106 @@ public void testNumericQueries() throws IOException {
     cleanMetadataStorage(records);
   }
 
+  @Test
+  public void testDateParser() throws IOException {
```

Comment:
this test should be part of `QueryParserTest`

Reply:
Also what would throw an IOException in this case?

Reply:
You're right, IOException was not being used. Moved the test and removed the IOException.